diff -ru rubygems.orig/lib/rubygems/commands/install_command.rb rubygems/lib/rubygems/commands/install_command.rb
--- rubygems.orig/lib/rubygems/commands/install_command.rb	2011-04-12 11:13:54.000000000 +0100
+++ rubygems/lib/rubygems/commands/install_command.rb	2011-04-12 11:14:49.000000000 +0100
@@ -19,8 +19,8 @@
 
   def initialize
     defaults = Gem::DependencyInstaller::DEFAULT_OPTIONS.merge({
-      :generate_rdoc     => true,
-      :generate_ri       => true,
+      :generate_rdoc     => false,
+      :generate_ri       => false,
       :format_executable => false,
       :test              => false,
       :version           => Gem::Requirement.default,
@@ -40,7 +40,7 @@
   end
 
   def defaults_str # :nodoc:
-    "--both --version '#{Gem::Requirement.default}' --rdoc --ri --no-force\n" \
+    "--both --version '#{Gem::Requirement.default}' --no-rdoc --no-ri --no-force\n" \
     "--no-test --install-dir #{Gem.dir}"
   end
 
diff -ru rubygems.orig/lib/rubygems/commands/update_command.rb rubygems/lib/rubygems/commands/update_command.rb
--- rubygems.orig/lib/rubygems/commands/update_command.rb	2011-04-12 11:13:54.000000000 +0100
+++ rubygems/lib/rubygems/commands/update_command.rb	2011-04-12 11:15:25.000000000 +0100
@@ -15,8 +15,8 @@
   def initialize
     super 'update',
           'Update the named gems (or all installed gems) in the local repository',
-      :generate_rdoc => true,
-      :generate_ri   => true,
+      :generate_rdoc => false,
+      :generate_ri   => false,
       :force         => false,
       :test          => false
 
@@ -37,7 +37,7 @@
   end
 
   def defaults_str # :nodoc:
-    "--rdoc --ri --no-force --no-test --install-dir #{Gem.dir}"
+    "--no-rdoc --no-ri --no-force --no-test --install-dir #{Gem.dir}"
   end
 
   def usage # :nodoc:
diff -ru rubygems.orig/lib/rubygems/install_update_options.rb rubygems/lib/rubygems/install_update_options.rb
--- rubygems.orig/lib/rubygems/install_update_options.rb	2011-04-12 11:13:54.000000000 +0100
+++ rubygems/lib/rubygems/install_update_options.rb	2011-04-12 11:17:58.000000000 +0100
@@ -110,7 +110,7 @@
   # Default options for the gem install command.
 
   def install_update_defaults_str
-    '--rdoc --no-force --no-test --wrappers'
+    '--no-rdoc --no-force --no-test --wrappers'
   end
 
 end
